docs/spec/api.md

```diff
@@ -2055,7 +2055,7 @@ The following headers will be returned with the response:
 
 |Name|Description|
 |----|-----------|
-|`Range`|Range indicating the current progress of the upload.|
+|`Range`|Range indicating the current progress of the upload. In 2.0.x offset is always zero since resumable push is not supported.|
```

Comment:
Where would you expect this to be documented.  Anyone looking at the spec may interpret this as being fully implemented by the registry. Should a section be added here noting gaps in the implementation or point to a document referencing known implementations.